src/webview/wkwebview/mod.rs

```diff
@@ -476,6 +476,10 @@ r#"Object.defineProperty(window, 'ipc', {
         let ns_window = window.ns_window() as id;
         let _: () = msg_send![ns_window, setContentView: webview];
 
+        // To prevent memory leak, we need to remove webview by script message handler.
+        let interop = NSString::new("interop");
+        let _: () = msg_send![manager, addScriptMessageHandler:webview name:interop];
```

Comment:
I feel like this should be the ipc [handler](https://github.com/tauri-apps/wry/blob/632d5439f8e1717cd86cb3d5bb49bb07a3d13545/src/webview/wkwebview/mod.rs#L302). The type of `addScriptMessageHandler` is `id<WKScriptMessageHandler>`. This is exact protocol WebViewDelegate has.